primefaces/src/main/resources/META-INF/resources/primefaces/core/core.ajax.js

```diff
@@ -315,7 +315,21 @@ if (!PrimeFaces.ajax) {
                     PrimeFaces.ajax.Utils.updateBody(content);
                 }
                 else if (id === PrimeFaces.ajax.RESOURCE) {
-                    $('head').append(content);
+                    // #11714 Iterate through each script and stylesheet tag in the content
+                    // checking if resource is already attached to the head and adding it if not
+                    $(content)
+                        .filter("link[href], script[src]")
+                        .each(function() {
+                            var $resource = $(this);
+                            var src = $resource.attr("href") || $resource.attr("src");
+                            var type = this.tagName.toLowerCase();
+
+                            // Check if script or stylesheet already exists and add it to head if it does not
+                            if (!$("head").find(type + '[src="' + src + '"], ' + type + '[href="' + src + '"]').length) {
```

Comment:
just cosmetics as bit more readable:
i would add this to a var `$("head").find(type + '[src="' + src + '"], ' + type + '[href="' + src + '"]')`
and better write:
if ($resources.length == 0)

Reply:
Good feedback.  Pushed.